src/elements/tabs/tabs.tsx

```diff
@@ -90,13 +91,17 @@ export const Tabs = ({
 }: TTabsProps) => {
   return (
     <TabsRoot value={value} onValueChange={onValueChange} {...props}>
-      <TabsList data-testid="tabs-list">
-        {values.map(({ label, value }) => (
-          <TabsTrigger data-testid="tabs-trigger" key={value} value={value}>
-            {label}
-          </TabsTrigger>
-        ))}
-      </TabsList>
+      <div className="flex justify-between">
+        <TabsList data-testid="tabs-list">
+          {values.map(({ label, value }) => (
+            <TabsTrigger data-testid="tabs-trigger" key={value} value={value}>
+              {label}
+            </TabsTrigger>
+          ))}
+        </TabsList>
+        {value === 'PULLS' &&
+          values.map(({ value, actions }) => <div key={value}>{actions}</div>)}
+      </div>
```

Comment:
Nope, this is very bad way. You are making a core component like a specific domain component.

Reply:
Sorry, I forgot that I had to modify that.